src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Read.Stream.cs

```diff
@@ -85,7 +86,104 @@ public static partial class JsonSerializer
             if (returnType == null)
                 throw new ArgumentNullException(nameof(returnType));
 
-            return ReadAllAsync<object>(utf8Json, returnType, options, cancellationToken);
+            return ReadAllUsingOptionsAsync<object>(utf8Json, returnType, options, cancellationToken);
+        }
+
+        private static ValueTask<TValue?> ReadAllUsingOptionsAsync<TValue>(
```

Comment:
Nit: keeping private methods above public APIs.

Reply:
Just saw I only addressed this for the Serialize APIs. Will make follow up for Deserialize APIs.